src/Async.js

```diff
@@ -40,7 +40,7 @@ const defaultProps = {
 	children: defaultChildren,
 	ignoreAccents: true,
 	ignoreCase: true,
-	loadingPlaceholder: 'Loading...',
+	loadingPlaceholder: '',
```

Comment:
This legend appears when the options are being loaded, or for example when you close a reference list,
please remove it only for the unique selector

Reply:
Yeah, I talked to UX team about that and they are expecting to do that change on all of the selects